Problem with compilation against proj master

I am trying to compile it from source. My libproj is compiled from master:

```
[ 31%] Building CXX object src/util/CMakeFiles/util.dir/moc_zuFile.cpp.o
cd /home/saber/repos/XyGrib/build/src/util && /usr/bin/c++   -I/home/saber/repos/XyGrib/build/src/util -I/home/saber/repos/XyGrib/src/util -I/usr/include/x86_64-linux-gnu/qt5 -I/usr/include/x86_64-linux-gnu/qt5/QtWidgets -I/usr/include/x86_64-linux-gnu/qt5/QtGui -I/usr/include/x86_64-linux-gnu/qt5/QtCore -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++ -I/usr/include/x86_64-linux-gnu/qt5/QtNetwork -I/usr/include/x86_64-linux-gnu/qt5/QtPrintSupport -I/usr/include/x86_64-linux-gnu/qt5/QtXml -I/usr/include/openjpeg-2.3 -I/usr/local/include -I/home/saber/repos/XyGrib/build/CMakeFiles/include -I/home/saber/repos/XyGrib/src/util/.. -I/home/saber/repos/XyGrib/src/util/../map  -Wall -fPIC   -std=gnu++11 -o CMakeFiles/util.dir/moc_zuFile.cpp.o -c /home/saber/repos/XyGrib/build/src/util/moc_zuFile.cpp
In file included from /home/saber/repos/XyGrib/src/util/../map/Projection.h:24,
                 from /home/saber/repos/XyGrib/src/util/../map/POI.h:34,
                 from /home/saber/repos/XyGrib/src/util/Settings.h:29,
                 from /home/saber/repos/XyGrib/src/util/Util.cpp:32:
/usr/local/include/proj_api.h:37:2: error: #error 'To use the proj_api.h you must define the macro ACCEPT_USE_OF_DEPRECATED_PROJ_API_H'
 #error 'To use the proj_api.h you must define the macro ACCEPT_USE_OF_DEPRECATED_PROJ_API_H'
  ^~~~~

```
I tried with:
`cmake -DACCEPT_USE_OF_DEPRECATED_PROJ_API_H=1 ..`

but with no luck.

--- a/src/map/Projection.h
+++ b/src/map/Projection.h
@@ -21,6 +21,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include <QObject>
 #include <cstdio>
 
+#ifndef ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
+#define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H
+#endif
 #include "proj_api.h"
 
 class Projection : public QObject

